Documentation/Installation/Install.rst

```diff
@@ -26,19 +26,24 @@ Execute Composer Create-Project
 
 At the root level of your web server, execute the following command:
 
+..  note::
+    The script here installs the latest TYPO3 v12.0. If you are looking for installation
+    instructions on the latest stable release, TYPO3 v11 LTS go to the :ref:`TYPO3 v11
```

Comment:
```suggestion
    instructions on the recent long-term release, TYPO3 v11 LTS go to the :ref:`TYPO3 v11
```

To make it clear what is meant. v12.0 is also "stable" in some kind but a sprint release.

Also would like "recent" or "current" instead of "latest".